features/update/activities/feature.go

```diff
@@ -0,0 +1,85 @@
+package activities
+
+import (
+	"context"
+	"time"
+
+	"go.temporal.io/features/features/update/updateutil"
+	"go.temporal.io/features/harness/go/harness"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/workflow"
+)
+
+const (
+	updateActivity = "updateActivity"
+	activityResult = 6
+	activityCount  = 5
+	done           = "done"
+	shutdownSignal = "shutdown_signal"
+)
+
+var Feature = harness.Feature{
+	Workflows:  Workflow,
+	Activities: SomeActivity,
+	Execute: func(ctx context.Context, runner *harness.Runner) (client.WorkflowRun, error) {
+		if reason := updateutil.CheckServerSupportsUpdate(ctx, runner.Client); reason != "" {
+			return nil, runner.Skip(reason)
+		}
+		run, err := runner.ExecuteDefault(ctx)
+		if err != nil {
+			return nil, err
+		}
+
+		handle, err := runner.Client.UpdateWorkflow(
+			ctx,
+			run.GetID(),
+			run.GetRunID(),
+			updateActivity,
+		)
+		runner.Require.NoError(err)
+
+		var result int
+		runner.Require.NoError(handle.Get(ctx, &result))
+		runner.Require.Equal(activityResult*activityCount, result)
+
+		runner.Require.NoError(runner.Client.SignalWorkflow(ctx, run.GetID(), run.GetRunID(), shutdownSignal, nil))
+		updateutil.RequireNoUpdateRejectedEvents(ctx, runner)
```

Comment:
Do we need this check here since `UpdateHandlerOptions` doesn't have a validator? Or is the intention to have this check for every sync. update test? I notice it was missing from the panic test case?

Reply:
I think my intent was to add it to all of them. That it's missing from the panic test is just an oversight - I'll add it.